cmd/uplink/cmd/mb.go

```diff
@@ -41,19 +41,27 @@ func makeBucket(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("Nested buckets not supported, use format sj://bucket/")
 	}
 
-	metainfo, _, err := cfg.Metainfo(ctx)
+	project, err := cfg.GetProject(ctx)
 	if err != nil {
+		fmt.Printf("Error setting up project uplink: %+v\n", err)
 		return err
 	}
+	defer func() {
+		err = project.Close()
+		if err != nil {
+			fmt.Printf("Error closing project uplink: %+v\n", err)
+		}
+	}()
 
-	_, err = metainfo.GetBucket(ctx, dst.Bucket())
+	_, _, err = project.GetBucketInfo(ctx, dst.Bucket())
```

Comment:
Maybe just create a bucket and see if `CreateBucket` returns an error. i think it should return something like "Bucket already exists". Just to avoid one request to satellite.

Reply:
I think at the moment `CreateBucket` will succeed, even the bucket already exists - it will just overwrite the existing bucket segment in pointerdb.

We need to finish the metainfo refactoring, and make it return an appropriate error if the bucket already exists.

Reply:
sounds good